src/main/frontend/search/db.cljs

```diff
@@ -27,8 +27,9 @@
      :page page
      :content result}))
 
+;; TODO: Do we want to pass repo here?
```

Comment:
This doesn't pass the repo to get-all-block-contents which just uses conn/get-conn. Should it?

Reply:
Yes, we can remove it.

Reply:
I left this alone for now as removing repo makes multiple upstream fns have confusing/unused repo arguments as well